assets/javascripts/discourse/connectors/user-summary-stat/sentiment.hbs

```diff
@@ -0,0 +1,16 @@
+{{#if
+  (and
+    this.siteSettings.ai_sentiment_enabled
+    @outletArgs.model.sentiment
+    this.showSentiment
+  )
+}}
```

Comment:
We can probably get rid of this wrapping `{{#if}}` and just move the logic inside the `shouldRender()` method:

```js
  static shouldRender(outletArgs, helper) {
    return helper.siteSettings.ai_sentiment_enabled && outletArgs.model.sentiment && helper.currentUser && helper.currentUser.staff;
  }
  ```

we can also avoid injecting `siteSettings` and `currentUser` services this way since already provided via `helper`

Reply:
:chef_kiss: